Dimik/even-odd-1/en.md

```diff
@@ -0,0 +1,39 @@
+# Even Odd 1 
+
+## Hint: You just need to handle last bit 
+
+### Solution 
+* for each test case take the number as a string 
+* Check the last character of the string 
+* make it integer 
+* If its divisible by 2
+  * its a even number so print 'even'
+* if its not divisible by 2 
+  * its a odd number so print 'odd'
```

Comment:
Lose the list and summarize it in a few sentences. (You shouldn't need much)
Additionally, you don't need to go all the way to taking the input as a string and checking the last character and all that.
The intended and easier approach would be to divide the number by 2 and checking the remainder. If the remainder is zero, then the number is even; otherwise, odd.